src/react/ISV/components/Modal/WelcomeModal.tsx

```diff
@@ -0,0 +1,192 @@
+  /*
+   We display the  welcome modal only if the following conditions are met:
+
+   1. No account exists in the platform
+   2. Storage Manager is logged in
+   3. Not already in the ISV config
+   4. The user skip it until the next login or login for the first time
+   5. No trial license modal displays
+   */
+  const isWelcomeModalEnabled =
+    isStorageManager &&
+    isZeroAccountCreated &&
+    !isAlreadyInConfigurationView &&
+    isNextLogin &&
+    !isTrialLicenseModalDisplayed;
+
+  if (!isWelcomeModalEnabled) {
+    return <></>;
+  }
+
+  return <ModalComponent />;
+};
+
+const ModalComponent = () => {
+  const [isOpen, setIsOpen] = useState<boolean>(true);
+  const [selectedISV, setSelectedISV] = useState<ISVCardConfig>(null);
+  const { useLinkOpener, useDeployedApps, useAuth } = useShellHooks();
+  const { openLink } = useLinkOpener();
+  const deployedApps = useDeployedApps();
+  const zenkoUI = deployedApps.find(
+    (app: { kind: string }) => app.kind === 'zenko-ui',
+  );
+
+  const currentApp =
+    deployedApps.find(
+      (app) =>
+        window.location.pathname.startsWith(app.appHistoryBasePath) &&
+        app.appHistoryBasePath !== '',
+    )?.kind ?? deployedApps.find((app) => app.appHistoryBasePath === '')?.kind;
+
+  const zenkoUIConfigurationView = useMemo(() => {
+    const view = selectedISV?.assistant
+      ? {
+          path: `/isv/configuration?platform=${selectedISV.id}`,
+          label: {
+            en: 'ISV Configuration',
+            fr: 'Configuration ISV',
+          },
+          module: './FederableApp',
+          scope: 'zenko',
+        }
+      : {
+          path: `/accounts`,
+          label: {
+            en: 'Accounts',
+            fr: 'Comptes',
+          },
+          module: './FederableApp',
+          scope: 'zenko',
+        };
+    return view;
+  }, [selectedISV]);
+
+  const configurationView = {
+    view: zenkoUIConfigurationView,
+    app: zenkoUI,
+    isFederated: true as const,
+  };
+
+  const user = useAuth();
+  const session_state = user?.userData?.original?.session_state;
+
+  const handleContinueClick = () => {
+    setIsOpen(false);
+    // If we are already in zenko-ui context, we can't use the openLink function.
+    // That's why we have to create a custom event, and listen to it to change the route.
+
+    if (currentApp === 'zenko-ui') {
+      const event = new CustomEvent('HistoryPushEvent', {
+        detail: {
+          path: configurationView.view.path,
+        },
+      });
+      window.dispatchEvent(event);
+    } else {
+      openLink(configurationView);
+    }
+  };
+
+  return (
+    <CustomModal
+      title={
+        <Stack direction="horizontal" gap="r8">
+          <Text variant="Large">Welcome to</Text>
+          <ArtescaLogo />
+          <Text variant="Large">ARTESCA - Connector Marketplace</Text>
+          <Text variant="Large" color="textSecondary">
+            Draft Concept
+          </Text>
```

Comment:
@Cuervino can we remove this "Draft concept" now ?

Reply:
Yes of course.